scan.go

```diff
@@ -238,6 +239,16 @@ func scan(pkgPatterns []string) ([]*unit.SourceUnit, error) {
 		// compare results in tests more easily.
 		pkg.Root = ""
 
+		if strings.HasPrefix(pkg.ImportPath, "_") {
+			wd, err := os.Getwd()
+			if err != nil {
+				return nil, err
+			}
+			pkg.ImportPath = repo + pkg.ImportPath[1+len(wd):]
```

Comment:
I just checked the CLI of `go list`, could not find an option to follow symlinks.

The reasoning is good, it just doesn't work so easily. ;-)

There is a bit more invasive option:
1. Use `go list -e ./...` (without `-json`) to just get the list of packages (or scan directories manually).
2. Prepare `.srclib-gopath` if necessary
3. Use http://golang.org/pkg/go/build/#Context.Import to get the package infos.